src/store/legal/index.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  createSlice, PayloadAction
+} from '@reduxjs/toolkit';
+
+export type Legal = {
+  contact?: string;
+  imprint?: string;
+  privacy?: string;
+};
+
+const initialState: Legal = {};
```

Comment:
We might want to set the defaults here.

Reply:
Defaults got moved from the component to the store.